doc/classes/AnimationNodeOneShot.xml

```diff
@@ -5,6 +5,27 @@
 	</brief_description>
 	<description>
 		A resource to add to an [AnimationNodeBlendTree]. This node will execute a sub-animation and return once it finishes. Blend times for fading in and out can be customized, as well as filters.
+		After setting the request and changing the animation playback, the one-shot node automatically clears the request on the next process frame by setting its [code]request[/code] value to [code]ONE_SHOT_REQUEST_NONE[/code].
```

Comment:
Use `[constant NAME]` instead of code markup to refer to constants with hyperlinks. (Same below.)